### PR TITLE
[MNOE-1166] add license based pricing

### PR DESCRIPTION
**Pull request by iseessel**

### CODE REVIEW

File: src/app/views/orders/order.html [hunk 1]
```diff
@@ -47,6 +47,22 @@
...
           </div>
+          <div class="row">
+            <div class="col-xs-6">
+              <div class="label-cell" translate>mnoe_admin_panel.dashboard.subscription.price</div>
+            </div>
+            <div class="col-xs-6">
+              <mnoe-product-pricing-price subscription="vm.subscription" selected-currency="vm.subscription.currency">
+            </div>
+          </div>
+          <div class="row" ng-show="vm.subscription.product_pricing.license_based">
```
Comment (adamaziz15) [inline src/app/views/orders/order.html]: Will this throw an error if there is no product_pricing attached to the subscription? Might want to use `vm.subscription.product_pricing?.license_based` here. If it does need to have it, this applies to everywhere else `vm.subscription.product_pricing` is used as well.

File: src/app/views/provisioning/order.html [hunk 2]
```diff
@@ -45,6 +45,29 @@
...
+            <label>{{ 'mnoe_admin_panel.dashboard.provisioning.order.licenses' | translate }}:
+              <input type="number" name="input" ng-model="vm.subscription.max_licenses" min="1" ng-required="vm.subscription.product_pricing.license_based">
+            </label>
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            <label>{{ 'mnoe_admin_panel.dashboard.provisioning.order.total' | translate }}:
+              <span ng-hide="vm.subscription.product_pricing.pricing_type == 'payg'" ng-repeat="price in vm.subscription.product_pricing.prices | filterBy: ['currency']: vm.orgCurrency" class="badge">
```
Comment (adamaziz15) [inline src/app/views/provisioning/order.html]: Should free plans be taken into account here as well?

File: src/app/views/provisioning/product-pricing-price.coffee [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@App.directive('mnoeProductPricingPrice', () ->
```
Comment (adamaziz15) [inline src/app/views/provisioning/product-pricing-price.coffee]: Might want to rename this to make it plural as the component will display all of the prices, not just one.

File: src/app/views/provisioning/product-pricing-price.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<span ng-hide="subscription.product_pricing.pricing_type == 'payg'">
```
Comment (adamaziz15) [inline src/app/views/provisioning/product-pricing-price.html]: Should free plans be taken into account here too?

File: src/app/views/orders/order.html (hunk 1 above)
Comment (iseessel) [reply src/app/views/orders/order.html]: Hmm... well I don't get any console errors when the product doesn't have a product_pricing.
It seems that angularjs takes care of this when its in a directive.

File: src/app/views/provisioning/order.html (hunk 2 above)
Comment (iseessel) [reply src/app/views/provisioning/order.html]: sure

File: src/app/views/provisioning/product-pricing-price.coffee (hunk 3 above)
Comment (iseessel) [reply src/app/views/provisioning/product-pricing-price.coffee]: sure

File: src/app/views/provisioning/product-pricing-price.html (hunk 4 above)
Comment (iseessel) [reply src/app/views/provisioning/product-pricing-price.html]: sure

**Pull request merged**